ext/spl/php_spl.c

```diff
@@ -413,6 +413,37 @@ static bool autoload_func_info_equals(
 }
 
 static zend_class_entry *spl_perform_autoload(zend_string *class_name, zend_string *lc_name) {
+	/*  classmap provides for classname => path resolution without needing to invoke a user function */
+	if (SPL_G(autoload_classmap)) {
+		zval * mf = zend_hash_find(SPL_G(autoload_classmap), lc_name);
```

Comment:
```suggestion
		zval *mf = zend_hash_find_deref(SPL_G(autoload_classmap), lc_name);
```
The entry could technically be a reference.